pkgs/servers/draupnir/update.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl common-updater-scripts nodePackages.node2nix gnused nix coreutils jq yarn yarn2nix
+
+set -euo pipefail
+
+echo "Getting versions..."
+latestVersion="$(curl -s "https://api.github.com/repos/Gnuxie/Draupnir/releases?per_page=1" | jq -r ".[0].tag_name" | sed 's/^v//')"
```

Comment:
Not by any means essential, but you could use `jq -r '.[0].tag_name | lstrimstr("v")'` and avoid the `sed`.